### PR TITLE
What is misspelled?

### PR DESCRIPTION
**Pull request by prestoncarman**

### CODE REVIEW

File: .github/actions/spelling/expect.txt [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+config
+Controlb
+cout
+cpp
+css
+CTORS
+cxx
+datetime
+ddl
+defualt
```
Comment (jgfoster) [inline .github/actions/spelling/expect.txt]: ?

File: .github/actions/spelling/expect.txt [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+QHG
+qhp
+qsf
+rb
+README
+REFIN
+REFINHIGH
+REFINLOW
+relpath
+requirments
```
Comment (jgfoster) [inline .github/actions/spelling/expect.txt]: ?

File: .github/actions/spelling/expect.txt [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+searchdata
+SEARCHENGINE
+seo
+setpoint
+showinitializer
+sideeffect
+sizeof
+Smath
+src
+Ssymbols
```
Comment (jgfoster) [inline .github/actions/spelling/expect.txt]: I can't find this in the repository. Is it part of `AMSsymbols`?

File: .github/actions/spelling/expect.txt [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+RTDINLOW
+rtf
+searchdata
+SEARCHENGINE
+seo
+setpoint
+showinitializer
+sideeffect
+sizeof
+Smath
```
Comment (jgfoster) [inline .github/actions/spelling/expect.txt]: Is this part of `AMSmath`?

File: .github/actions/spelling/expect.txt [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+Ssymbols
+startuml
+stderr
+strlen
+stylesheet
+subdir
+subgrouping
+Sukachevin
+svg
+swtiching
```
Comment (jgfoster) [inline .github/actions/spelling/expect.txt]: ?

**Review by jgfoster — CHANGES_REQUESTED** [review]
See notes...

File: .github/actions/spelling/expect.txt (hunk 3 above)
Comment (prestoncarman) [reply .github/actions/spelling/expect.txt]: All the "words" listed in this file were suggested by the spelling action. It seems that maybe installed libraries or other sources outside our code are included. Although I am not sure where they get included.

Comment (prestoncarman) [reply .github/actions/spelling/expect.txt]: Addressed by removing configuration file for Doxygen (./Doxyfile) from spell check.

File: .github/actions/spelling/expect.txt (hunk 4 above)
Comment (prestoncarman) [reply .github/actions/spelling/expect.txt]: Same as above comment.

**Comment by prestoncarman** [comment]
The new exception file is much more reasonable file size and hopefully this time does not have unwanted misspellings. :-) You will see the change also include several fixes for spelling errors.

**Review by jgfoster — APPROVED** [review]
LGTM

**Pull request merged**